Missionframework/presets/blufor/rhs_afrf.sqf

```diff
@@ -0,0 +1,332 @@
+/*
+Needed Mods:
+- RHS AFRF
+
+Optional Mods:
+- None
+*/
+
+/* - Support classnames.
+Each of these should be unique, the same classnames for different purposes may cause various unpredictable issues with player actions. Or not, just don't try!	*/
+FOB_typename = "Land_Cargo_HQ_V1_F";									// This is the main FOB HQ building. 																									Default is "Land_Cargo_HQ_V1_F".
+FOB_box_typename = "B_Slingload_01_Cargo_F";							// This is the FOB as a container. 																										Default is "B_Slingload_01_Cargo_F".
+FOB_truck_typename = "rhs_gaz66_r142_msv";								// This is the FOB as a vehicle.																										Default is "B_Truck_01_box_F".
+Arsenal_typename = "O_supplyCrate_F";									// This is the virtual arsenal as portable supply crates.  																				Default is "B_supplyCrate_F".
+Respawn_truck_typename = "rhs_gaz66_ap2_msv";							// This is the mobile respawn (and medical) truck. 																						Default is "B_Truck_01_medical_F".
+huron_typename = "RHS_Mi8mt_vvs";										// This is Spartan 01, a multipurpose mobile respawn as a helicopter. 																	Default is "B_Heli_Transport_03_unarmed_F".
```

Comment:
If I remember correctly there was an "issue" that the Mi8 can't lift the FOB Box. Have you tested this?

Reply:
It can't lift it via _vanilla_ slingload. It works fine when using advanced rapelling tho.

If this [config explorer](https://configs.arma3.ru/154.133741-rhs/configfile/CfgVehicles/RHS_Mi8mt_vvs.html) is not outdated then Mi8 can lift 2700 mass, FOB container is set to 3000 mass in "startgame.sqf" and in "do_repackage_fob.sqf". I will test if It will be able to lift it via _vanilla_ system later.

Reply:
Guess we could maybe change the mass of the box to 2600 later on. But it should still be too much for a ghost hawk or equivalent.

Reply:
Black Hawk and Ghosthawk can slingload 4000 kg so even currently they can lift FOB container, after lowering weight of container to 2700, Mi8 was also able to lift it. 

I think container mass should be moved to variable in preset files.

Reply:
Not to the preset files. That's too much.
I'll maybe add a short config lookup for the "max lift mass" of the spartan helicopter, which is then lowered by 100 and then applied to the FOB container mass. Or generally lower the mass to 2500 for example.
Currently in preparations for moving to a new flat, so not that much commits now from my side.